scripts/xtensa-build-zephyr.py

```diff
@@ -20,8 +20,7 @@
 
 # Constant value resolves SOF_TOP directory as: "this script directory/.."
 SOF_TOP = pathlib.Path(__file__).parents[1].resolve()
-# Default value may be overriten by -p arg
-west_top = pathlib.Path(SOF_TOP, "zephyrproject")
+west_top = pathlib.Path(SOF_TOP, "..").resolve()
 default_rimage_key = pathlib.Path("modules", "audio", "sof", "keys", "otc_private_key.pem")
```

Comment:
please change rimage and rimage key path accordingly.
Note that after #6004 get merged, sof is decoupled with zephyr, the soft link to zephyr/samples/audio/sof is not needed. please rebase.

Reply:
Link is still needed this is not a rebase issue.
PR #6004 decouples SOF from Zephyr in a sense however from developers point of view it does not.
Why?
Because when we make changes to Zephyr code and want to submit a PR to Zephyr upstream we run twister tests which are part of the Zephyr CI - which includes building SOF application.
And Zephyr still expects to SOF project to be checked out in modules/audio/sof .
To sum up:

- yes you may build now SOF without this symlink
- however this symlink should stay for now for developers convenience to run twister tests (`./zephyr/scripts/twister -p <board_name>`)
- I will rebase in a moment

Reply:
@aborisovich, I checked your latest version and it seems that the symlink is missing:
![image](https://user-images.githubusercontent.com/22468760/182122840-848ad668-5065-4581-9268-4a1e9d91d539.png)
I did not find where we create the symlink, please check.

Reply:
The `../../../sof` symbolic link was critical before this PR when using `sof/zephyrproject/west.yml`. Now that `sof/zephyrproject/west.yml` is being replaced by this new `sof/west.yml` manifest, I understand the symlink has become only a "nice to have" to run `twister` when using `sof/west.yml`. The symlink can still be easily created manually by anyone who needs it. So I recommend not to delay the transition to the new manifest just waiting for this symlink issue to be fixed.